### PR TITLE
Allow user parameters for shuffle

### PR DESCRIPTION
**Pull request by nishnash54**
Implement shuffle parameter for the following features
- [x] Data split `train_test_split()`
- [x] Cross validation `KFold()` + `StratifiedKFold`

Added user parameters
- `data_split_shuffle`: for train test split
- `folds_shuffle`: for cross validation
Maintains compatibility as `default` values set to match that of `sklearn`

Fix issues: #12 #68

### CODE REVIEW

**Comment by pycaret** [comment]
@nishnash54 Also please can you submit the request to dev-1.0.1 branch.

File: classification.py [hunk 1]
```diff
@@ -1262,8 +1263,8 @@ def highlight_max(s):
         sample_size = input("Sample Size: ")
         
         if sample_size == '' or sample_size == '1':
-            
-            X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=1-train_size, stratify=y, random_state=seed, shuffle=data_split_shuffle)
+            stratify = y if data_split_shuffle else None
```
Comment (nishnash54) [inline classification.py:1266]: Yes, I you have a look at the changes in the latest commit, I have implemented option 1.
If the `shuffle` parameter is set to `True`, the `stratify` is automatically set to `None` to work with the default `sklearn` behavior. This is also updated in the documentation.

**Comment by pycaret** [comment]
Merging now.

Thanks everyone.

**Comment by ngupta23** [comment]
Is this working? I am running an example on the dev branch and it is complaining about 
"name 'folds_shuffle' is not defined". Should this be passed as a parameter to every function in the code or made into a global variable?

**Comment by pycaret** [comment]
Yes. This is not working. I am currently working on regression.py and dev branch. Making changes and fixing issues introduced by changes. Will commit by tomorrow.

**Comment by ngupta23** [comment]
Thank you. That will be great!.
